dynamiqs/utils/vectorization.py

```diff
@@ -223,6 +223,10 @@ def slindbladian(H: ArrayLike, jump_ops: ArrayLike) -> Array:
     Note:
         This superoperator is also sometimes called *Liouvillian*.
 
+    Note:
+        See also [`dq.lindbladian`][dynamiqs.utils.quantum_utils.general.lindbladian] to
+        access the Lindbladian only through matrix-vector product.
```

Comment:
Also, if should say "matrix-matrix product" instead of "matrix-vector product" I think.